index.bs

```diff
@@ -4592,7 +4718,7 @@ object">WebVTT region objects</a> must take their initial values.</p>
 then they must be interpreted as defined in the next section.</p>
 
 
-<h4 id=css-extensions>CSS extensions</h4>
+<h3 id=css-extensions>Processing CSS extensions</h3>
```

Comment:
I think this should be a subsection of the "CSS extensions" section above, so the authoring intro and the processing model are together.